resultsdbupdater/utils.py

```diff
@@ -182,66 +182,67 @@ def ci_metrics_post_to_resultsdb(msg):
 
 def cips_post_to_resultsdb(msg):
     session = retry_session()
+
     # define variables from cips.json
     msg_id = msg['headers']['message-id']
-    ci_type = msg['headers']['ci_type']
+    type = msg['headers']['type']
```

Comment:
Please don't use `type` since it overshadows a builtin Python function

Reply:
Can you suggest an alternative, @mprahl?  Maybe `type_`?

Reply:
Keeping it as `build_type` works too.

Reply:
I have commented the same in "FACTORY 2.0 CI UMB messages" google document.

Reply:
This doesn't need discussion since the problem is the variable name in Python not the key in the message header.